_posts/2020-02-21-status-update.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "21st Feb 2020: IAC Team Status Update"
+author: dev_el_ops
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - approved
+  - litmus
+  - powershell
+  - sqlserver
+  - vsphere
+  - modulebuilder
+  - stdlib
+---
+
+The IAC team has been working on a lot of interesting things over the past week.
+Here's a brief overview of what we've been up to:
+
+## Updates to Puppet Modules
+
+We've released only one module since last week:
+
+- [puppetlabs/java](https://forge.puppet.com/puppetlabs/java) (v6.2.0)
+
+## Community Contributions
+
+Thanks to [@baurmatt](https://github.com/baurmatt) for his change to the concat module,
+[adding support for Deferred function in concat fragment](https://github.com/puppetlabs/puppetlabs-concat/pull/627).
+
+## Approved Modules
+
+This week we happily approved two new modules:
```

Comment:
There's 3 listed

Reply:
there's not?

Reply:
This week we happily approved two new modules